sdk/communication/communication-chat/src/generated/src/chatApiClientContext.ts

```diff
@@ -6,13 +6,10 @@
  * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
-import * as coreHttp from "@azure/core-http";
+import * as coreClient from "@azure/core-client";
 import { ChatApiClientOptionalParams } from "./models";
 
-const packageName = "azure-communication-chat";
-const packageVersion = "1.1.0-beta.2";
```

Comment:
looks like another string to be removed from `constantPaths` inside package.json